NEWS.md

```diff
@@ -1,6 +1,32 @@
 shinydashboard 0.5.3.9000
 =========================
 
+This is a maintenance release of Shinydashboard, mostly aimed at fixing bugs and introducing minor features. 
+
+## Full changelog
+
+### New features
+
+* Address [#179](https://github.com/rstudio/shinydashboard/issues/179) support for bookmarking the expanded/collapsed state of the whole sidebar. (commit [#e71c93f](https://github.com/rstudio/shinydashboard/pull/199/commits/e71c93fa7a71f229e725efd4a7867e431cd57679))
```

Comment:
For commits, you don't need the `#`, so instead of `#e71c93f`, it would be `e71c93f`.